### ISSUE TITLE
Updates to work with newer versions?

### ISSUE THREAD

**Issue by null-zero** [Ocelot5836/Pop-Out-Chat#5]
Any chance this could be updated to 1.17/ 1.17.1?

**Comment by w1nst0nDEV** [comment]
Vouch ^

**Comment by SkyNetCloud** [comment]
I don't think this going to go pass 1.16 unless this dev has the time to rewrite most of the code cuz after 1.17 everything breaks, I'm pretty sure.
